examples/simple_example.rs

```diff
@@ -65,36 +64,53 @@ fn tracer() -> Box<Tracer> {
     }
 }
 
+/// Prints the addresses of the first `qty` blocks in a trace along with it's name and
+/// computation result.
+fn print_trace(trace: &Box<Trace>, name: &str, result: u32, qty: usize) {
```

Comment:
This feels like a debug function to me? And should it be printing directly or building a string and returning it to the user? It feels a bit odd to pass a `name` to a function that prints it straight out to stdout.

Reply:
Doh, ignore the "debug" comment -- I missed that this as an example :/

Reply:
Yeah, it's an example. Just doing something with the trace to offer the user some evidence that we actually did trace something :)